src/modules/sub-items/components/bulk-move/bulk.move.btn.js

```diff
@@ -0,0 +1,138 @@
+
+import { bulkMoveLocations } from './../../services/bulk.service.js';
+import Icon from '../../../common/icon/icon.js';
+
+class BulkMoveButton extends PureComponent {
+    constructor(props) {
+        super(props);
+
+        this.onBtnClick = this.onBtnClick.bind(this);
+        this.closeUdw = this.closeUdw.bind(this);
+        this.onUdwConfirm = this.onUdwConfirm.bind(this);
+
+        this.state = {
+            isUdwOpened: false,
+        };
+    }
+
+    componentDidMount() {
+        this.udwContainer = document.getElementById('react-udw');
+    }
+
+    onBtnClick() {
+        this.toggleUdw(true);
+    }
+
+    bulkMove(location) {
+        const { restInfo, selectedItems: itemsToMove } = this.props;
+        const contentsToMove = itemsToMove.map((item) => item.location);
+
+        bulkMoveLocations(restInfo, contentsToMove, location._href, this.afterBulkMove.bind(this, location));
+    }
+
+    afterBulkMove(location, movedLocations, notMoved) {
+        const { removeItemsFromList } = this.props;
+        const movedLocationsIds = new Set(movedLocations.map((location) => location.id));
+
+        removeItemsFromList(movedLocationsIds);
+
+        if (notMoved.length) {
+            const message = Translator.trans(
+                /*@Desc("You do not have permission to move at least 1 of the selected content item(s). Please contact your Administrator to obtain permissions.")*/ 'bulk_move.error.message',
+                {},
+                'sub_items'
+            );
+
+            window.eZ.helpers.notification.showErrorNotification(message);
+        }
+
+        if (movedLocations.length) {
+            const message = Translator.trans(
+                /*@Desc("The selected content item(s) have been sent to <u>%location_name%</u>")*/ 'bulk_move.success.message',
+                { location_name: location.ContentInfo.Content.Name },
+                'sub_items'
+            );
+
+            window.eZ.helpers.notification.showSuccessNotification(message);
+        }
+    }
+
+    toggleUdw(show) {
+        this.setState(() => ({
+            isUdwOpened: show,
+        }));
+    }
+
+    closeUdw() {
+        this.toggleUdw(false);
+    }
+
+    onUdwConfirm([selectedLocation]) {
+        this.closeUdw();
+        this.bulkMove(selectedLocation);
+    }
+
+    renderUdw() {
+        const { isUdwOpened } = this.state;
+
+        if (!isUdwOpened) {
+            return null;
+        }
+
+        const UniversalDiscovery = window.eZ.modules.UniversalDiscovery;
+        const { restInfo, parentLocationId, selectedItems, udwConfig } = this.props;
+        const selectedItemsLocationsIds = selectedItems.map((item) => item.location.id);
+        const excludedMoveLocations = [parentLocationId, ...selectedItemsLocationsIds];
+        const title = Translator.trans(/*@Desc("Choose location")*/ 'udw.choose_location.title', {}, 'sub_items');
+        const udwProps = {
+            ...udwConfig,
```

Comment:
The `udwConfig` should override the configuration from JS. This is the convention we use. The backend configuration has the priority.